From 1999d2aef9e7fde3585d3146952fc555f990287e Mon Sep 17 00:00:00 2001
From: builder <builder@localhost.localdomain>
Date: Wed, 29 Apr 2026 22:07:07 +0000
Subject: [PATCH] middleware: cleanup object files after install

---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index 1762b62..693bd23 100644
--- a/Makefile
+++ b/Makefile
@@ -95,6 +95,7 @@ endif
 	# copy mw lib
 	@cp -a lib/*.so*  $(DESTDIR)/usr/lib
 	@cp -a lib/3rd/*.so*  $(DESTDIR)/usr/lib/3rd
+	@find ./ -name '*.o' ! -name 'sample_common_*.o' -print0 | xargs -0 rm -f
 
 uninstall:
 	@rm $(DESTDIR) -rf
-- 
2.43.0

