From 481e6c1193b3bc221ff1caca69308423d2c88871 Mon Sep 17 00:00:00 2001
From: scpcom <scpcom@gmx.de>
Date: Sun, 18 May 2025 03:39:24 +0200
Subject: [PATCH] middleware: 3rdparty: opencv: cleanup build after install

---
 3rdparty/opencv/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/3rdparty/opencv/Makefile b/3rdparty/opencv/Makefile
index 5c01386..edbd897 100644
--- a/3rdparty/opencv/Makefile
+++ b/3rdparty/opencv/Makefile
@@ -190,6 +190,7 @@ $(TRD_LIB_PATH)/libopencv_core.a: $(TRD_LIB_PATH)/libopencv_core.so
 	@cd $(TRD_TMP_PATH)/opencv/build_static ; $(OPENCV_CMAKE_ENV) make install
 	@tar -C $(TRD_TMP_PATH)/opencv/output -cf $(TRD_TAR_PATH)/opencv_aisdk.tar.gz include lib share
 	@rsync -avpPxH $(TRD_TMP_PATH)/opencv/output/lib/libopencv*.a $(TRD_LIB_PATH)/
+	@rm -rf $(TRD_TMP_PATH)/opencv
 
 build_3rdparty_module: $(TRD_LIB_PATH)/libopencv_core.a
 
-- 
2.34.1

